XamlAnimatedGif.Shared/Decoding/GifGraphicControlExtension.cs

```diff
@@ -47,7 +51,8 @@ internal override GifBlockKind Kind
             DisposalMethod = (GifFrameDisposalMethod) ((packedFields & 0x1C) >> 2);
             UserInput = (packedFields & 0x02) != 0;
             HasTransparency = (packedFields & 0x01) != 0;
-            Delay = BitConverter.ToUInt16(bytes, 2) * 10; // milliseconds
+            Delay = BitConverter.ToUInt16(bytes, 2);
```

Comment:
Could you introduce a local variable here? It's a bit weird to have `Delay` represent tens of milliseconds, then milliseconds...